crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -3317,13 +3317,20 @@ impl AuthorityPerEpochStore {
                             "Received RandomnessDkgMessage from {:?}",
                             authority.concise()
                         );
-                        match bcs::from_bytes(bytes) {
-                            Ok(message) => randomness_manager.add_message(authority, message)?,
-                            Err(e) => {
-                                warn!(
+                        if bytes.len() > MAX_RANDOMNESS_DKG_MESSAGE_SIZE {
+                            warn!(
+                                "Ignoring RandomnessDkgMessage from {:?} because it is too large",
+                                authority.concise()
+                            );
+                        } else {
+                            match bcs::from_bytes(bytes) {
+                                Ok(message) => randomness_manager.add_message(authority, message)?,
```

Comment:
actually, i should have noticed this in an earlier review - can the `?` here end consensus processing early if a byzantine peer sends a bad message?